### DISCUSSION TITLE
Не могу убрать лишние поля

### DISCUSSION THREAD

**Discussion by iloaq** [woocommerce/woocommerce#44136] [Checkout Flow]
Нужно просто убрать поля лишние  и не получается

**Comment by opr** [comment]
Hi @iloaq thanks for opening this issue - unfortunately I can't speak Russian, so I used Google Translate to help me understand your comment.

> You just need to remove unnecessary fields and it doesn’t work

I think you might be talking about removing Checkout fields which is currently the topic of a discussion (https://github.com/woocommerce/woocommerce/discussions/42996). Please comment there with any input, and be sure to follow that discussion to keep up with our development on the feature.

Thanks!
